js/generation.js

```diff
@@ -1,8 +1,28 @@
+/*global
+  MimeMapper
+ */
+
 (function(exports) {
   'use strict';
 
+  function proposeDownload(blob) {
+    console.log('proposeDownload');
+    var url = window.URL.createObjectURL(blob);
+    var a = document.createElement('a');
+    a.href = url;
+    a.download = 'app.zip';
+    a.hidden = true;
+    document.body.appendChild(a);
+    a.click();
+    a.remove();
+    window.URL.revokeObjectURL(blob);
+  }
```

Comment:
on devices without the "developer mode" we download the zip file as well -- not really useful I admit